ansible/roles/delayed_job/README.md

```diff
@@ -5,5 +5,5 @@
 ### Optional
 
 * `delayed_job_queues`
-  The delayed job queues so the upstart script can start each one.
-  If this is not set, then the upstart script will start delayed\_job without specifying any queue and it will run all of your jobs.
+  The delayed job queues to process.
+  Defaults to: ["default"]
```

Comment:
Should some of this config move to a generic "job queue" list instead, since we need it defined regardless of what's processing the jobs